src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Find your way with field data in the Web Vitals extension
+subhead: The Web Vitals extension now shows you where your local experiences are in relation to
+  real user experiences in the field.
+description: The Web Vitals extension now shows you where your local experiences are in relation
+  to real user experiences in the field.
+authors:
+  - rviscomi
+date: 2021-06-30
+hero: image/STd8eW8CSiNp5B1bX0R6Dww2eH32/KwcWeCZ6LoIfTLRgVxp4.png
+alt: The Milky Way galaxy with a pin labeled You Are Here
+tags:
+  - blog
+  - web-vitals
+  - chrome-ux-report
+  - tools
+---
+
+[Core Web Vitals](https://web.dev/vitals) are the most important metrics you should be measuring to
+understand your users' experiences, because when users have good experiences,
+[good things happen](https://wpostats.com/)! The [Web Vitals extension](https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma?hl=en)
+for Chrome is one tool in the [Web Vitals toolbox](https://web.dev/vitals-tools/) that focuses on
+giving you performance data about the pages you visit as you browse the web. But you may be
+wondering, "how typical are user experiences like mine?" The last thing we want to do is perpetuate
+the myth that _if it's fast for me, it's fast for thee_. So we're launching version 1.0 of the
+Web Vitals extension with a new feature to integrate real-user data from the
+[Chrome UX Report](https://web.dev/chrome-ux-report/) (CrUX) with your local Core Web Vitals
+experiences, putting them in the greater context of how other users have experienced the same pages
+and websites. It comes with a sparkly new UI and I'm excited to show you how it works.
+
+{% Video
+  src="video/STd8eW8CSiNp5B1bX0R6Dww2eH32/EFeVK6dAwBZWzKhqBd1Y.mp4",
+  height="493",
+  width="800",
+  controls="true",
+  loop="true",
+  muted="true",
+  preload="auto",
+  class="w-screenshot"
+%}
+
```

Comment:
I suggest adding a subheading here, and for every other distinct section. This could be as simple as "CrUX and field data", but it will make the post easier to scan, and easier for readers to go back to specific sections.

Reply:
You placed them right, I probably lost my place in the file. Headings have to be sequential though, so H2. I'll change them.